lib/FreeRTOS/freertos-plus-mqtt/freertos_mqtt_agent.h

```diff
@@ -133,6 +138,14 @@ typedef void (* CommandCallback_t )( void * pCmdCallbackContext,
  * @param[in] pMqttAgentContext The context of the MQTT agent.
  * @param[in] packetId The packet ID of the received publish.
  * @param[in] pPublishInfo Deserialized publish information.
+ *
+ * @note As this callback is run in the task context of the MQTT agent, a command
+ * SHOULD NOT be queued to the agent from within the callback. If the application
+ * sequence wants to queue a command as an effect of executing this callback, this
+ * callback should notify a different task to enqueue commands to the MQTT agent.
+ *
+ * @note The context passed to the callback through the @p pMqttAgentContext parameter
+ * MUST remain in scope throughout the period that the agent task is running.
```

Comment:
This note should be part of MQTTAgent_Init API.

Reply:
Added